config/nrfconnect/chip-module/Kconfig

```diff
@@ -459,4 +459,10 @@ config CHIP_ENABLE_BDX_LOG_TRANSFER
 	help
 	  Enables the BDX protocol for diagnostics log transfer purposes.
 
+config CHIP_SYSTEM_PACKETBUFFER_POOL_SIZE
+  int "Packet buffer pool size"
+  default 11
+  help
+    Total number of packet buffers.
```

Comment:
I wonder if we can describe it a little bit more. For example, describe the purposes for which those packet buffers will be used.